server/index.js

```diff
@@ -3,7 +3,7 @@ let app = express();
 
 app.use(express.static(__dirname + '/../client/dist')); 
 
-app.get(`/companies/${companyName}`, function(req, res) {
+app.get('/companies/name', function(req, res) {
```

Comment:
Katie, does this work for any company name? consider using '/companies/:name' to make name optional and usable as a variable. If it works, feel free to discard this.